src/app/desktop/windows/terminal/terminal-states.ts

```diff
@@ -177,14 +182,16 @@ export class DefaultTerminalState extends CommandTerminalState {
     'chaozz': {
       executor: () => this.terminal.outputText('"mess with the best, die like the rest :D`" - chaozz'),
       description: ''
-    }
+    },
```

Comment:
Is this syntax correct?

Reply:
I think so...

Reply:
I think the comma is too much, isn't it?

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Trailing_commas
You're right.